orga/mirage/config.js

```diff
@@ -101,6 +101,9 @@ function routes() {
   });
 
   this.patch('/memberships/:id');
+  this.patch('/organizations/:id/me', () => {
```

Comment:
Je sais que c'est loin d'être le cas partout dans ce fichier, mais être plus explicite sur quel `id` il s'agit rend la compréhension plus rapide, d'autant qu'ici il est aussi question de `me` donc en lecture rapide on se demande s'il s'agit d'un `userId` ou d'un `organizationId` : 
```suggestion
  this.patch('/organizations/:organizationId/me', () => {
```